web/src/main/java/se/inera/intyg/webcert/web/service/facade/internalapi/service/GetTextsForCertificateService.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2023 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.webcert.web.service.facade.internalapi.service;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.stereotype.Service;
+import se.inera.intyg.common.support.facade.model.CertificateText;
+import se.inera.intyg.common.support.modules.registry.IntygModuleRegistry;
+import se.inera.intyg.common.support.modules.registry.ModuleNotFoundException;
+
+@Service
+public class GetTextsForCertificateService {
+
+    private final IntygModuleRegistry intygModuleRegistry;
+
+    public GetTextsForCertificateService(IntygModuleRegistry intygModuleRegistry) {
+        this.intygModuleRegistry = intygModuleRegistry;
+    }
+
+    public List<CertificateText> get(String type, String typeVersion) {
+        List<CertificateText> texts = new ArrayList<>();
+        try {
+            texts.add(intygModuleRegistry.getModuleApi(type, typeVersion).getPreambleForCitizens());
```

Comment:
I might be misunderstanding you but do we really want to throw an error if the description is not found? A missing text doesn't feel like something we want to invalidate the getCertificate response over. 

There's no possibility that the moduleApi won't have the function getPreambleForCitizens because then the application won't build. If we get a description which is null or empty this might be the desired behaviour and not something we should throw an error for.

Reply:
My intital thought was that is was a good idea to throw an IlligalStateException just to clearly state that we always expect a implementationen of the function, but it might be redundant. I just thought of it as a "safety net". 

But since not all certificates will have a premable text, maybe its not such a good idea after all.

In that case, the only error we expect is that the module-api was not found, and a separate function for getting the module-api may be redundant.